packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -648,12 +648,11 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 		 * @returns {undefined}
 		 */
 		cancelAnimation = () => {
+			this.stop();
```

Comment:
I think we would want `this.stop` to be controlled by `MarqueeDecorator` when `this.sync` is true and `this.context.cancel` eventually dispatches `this.stop`